code/modules/paperwork/photocopier.dm

```diff
@@ -120,7 +120,7 @@
 		else if(doccopy)
 			for(var/i = 0, i < copies, i++)
 				if(toner > 5 && !busy && doccopy)
-					new /obj/item/documents/photocopy(src, doccopy)
+					new /obj/item/documents/photocopy(loc)

```

Comment:
ha ha ha yes, otherwise you can photocopy documents endlessly for infinite points, considering the cost of toner